lib/src/migrators/calc_interpolation.dart

```diff
@@ -37,19 +37,30 @@ class _CalculationInterpolationVisitor extends MigrationVisitor {
     const calcFunctions = ['calc', 'clamp', 'min', 'max'];
     final interpolation = RegExp(r'\#{\s*[^}]+\s*}');
     final hasOperation = RegExp(r'[-+*/]+');
+    final isVarFunc = RegExp(
+        r'var\(#{[a-zA-Z0-9#{$}-]+}\)|var\(\-\-[a-zA-Z0-9\$\#\{\}\-]+\)');
     if (calcFunctions.contains(node.name)) {
       for (var arg in node.arguments.positional) {
         var newArg = arg.toString();
-        for (var match in interpolation.allMatches(arg.toString())) {
-          var noInterpolation =
-              match[0].toString().substring(2, match[0].toString().length - 1);
+        var varFuncArgs = isVarFunc.allMatches(newArg);
+        if (varFuncArgs.isNotEmpty) {
+          newArg = newArg.replaceAll(isVarFunc, 'var()');
+        }
+
+        for (var match in interpolation.allMatches(newArg)) {
+          String matchRegex = match[0].toString();
+          var noInterpolation = matchRegex.substring(2, matchRegex.length - 1);
           if (hasOperation.hasMatch(noInterpolation)) {
             noInterpolation = '(' + noInterpolation + ')';
           }
-          newArg = newArg
-              .toString()
-              .replaceAll(match[0].toString(), noInterpolation);
+          newArg = newArg.toString().replaceAll(matchRegex, noInterpolation);
         }
+
+        for (var match in varFuncArgs) {
+          String matchRegex = match[0].toString();
```

Comment:
Factoring this out into a local variable isn't necessary (especially since it's only used once here). My previous comment was saying to use the [non-null assertion operator (`!`)](https://dart.dev/language/operators#other-operators) instead of `toString`, since `match[0]` will never be null, it's just typed as `String?`.

Reply:
Ohh gotcha, I misread the comment sorry about that. Thanks ! :)